### ISSUE TITLE
Update example parameters after trainer changes

### ISSUE THREAD

**Issue by crstngc** [lanl/RMC#10]
The jax key update was misplaced in the training loop, causing no real data shuffling when training. Therefore, training hyperparameters for the examples must be checked and updated.
